upb/mini_table.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2009-2021, Google LLC
```

Comment:
probably want to update all of these to 2022 at some point

Reply:
Done.